src/components/reusable/tag/tag.scss

```diff
@@ -0,0 +1,219 @@
+@use '../../../common/scss/global.scss';
+@use '../../../common/scss/mixins.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+}
+
+.tags {
+  @include typography.type-ui-03;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  border-radius: 4px;
+  gap: 4px;
+  padding: 0px 4px 0px 4px;
```

Comment:
In the design, It looks like left/right padding should be 12px for medium size, and 6px for small size. 0px right padding for filter variant because of the X button.